x-pack/plugins/security_solution/public/detections/components/rules/query_preview/custom_histogram.test.tsx

```diff
@@ -51,7 +51,7 @@ describe('PreviewCustomQueryHistogram', () => {
 
     expect(wrapper.find('[data-test-subj="queryPreviewLoading"]').exists()).toBeTruthy();
     expect(
-      wrapper.find('[data-test-subj="queryPreviewCustomHistogram"]').at(0).prop('subtitle')
+      wrapper.find('[dataTestSubj="queryPreviewCustomHistogram"]').at(0).prop('subtitle')
```

Comment:
@yctercero I tried to make as few changes to these tests as possible to maintain behavior while allowing a custom `data-test-subj` on each histogram; see 471595f47eef3486fcc99d3f5b39767d4e5a54de for details

Reply:
Looks great!